### PR TITLE
ResetPassword Screen

### PR DESCRIPTION
**Pull request by Nap1400**
image

### CODE REVIEW

**Comment by Nap1400** [comment]
I know this is wrong, I need to fix the formatting
image

**Review by Josetf2396 — APPROVED** [review]

**Comment by lasharela** [comment]
We have to change Screen color through navigation theming

**Review by lasharela — APPROVED** [review]

**Comment by Dulaya** [comment]
The biggest issue is that your PR is behind the main branch by many merges:

- Add Container to your ResetPassword.view.tsx
   - Make sure you add the background image
- Okay to remove `TResetPasswordProps` from ResetPassword.type.ts if you're not using it. Make sure it doesn't break your app:

                  export type TResetPasswordProps = StackScreenProps<
                    TAuthNavigationRoutes,
                    EAuthNavigationRoutes.RESET_PASSWORD
                  >;
- Don't forget to add shadows to the TextInput and Button. See Miguel's or Charlie's screens for examples.
- Make sure you remove **"@types/react": "^18.0.8"**, from Package.json, because it'll give unnecessary type errors. We're type-checking React Native components. We're not type-checking React components.
- Button radius should be more rounder. 
- Navigation looks good. 
- Everything else looks good to me. ✔️

File: mobile/src/screens/resetPassword/ResetPassword.view.tsx [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+import { styles } from "./ResetPassword.style";
+import { TResetPasswordViewProps } from "./ResetPassword.type"
+
+export const ResetPasswordView: FC<TResetPasswordViewProps> = ({
+    title,
+    subtitle,
+    press,
+}) => {
+    return(
+        <View>
```
Comment (lasharela) [inline mobile/src/screens/resetPassword/ResetPassword.view.tsx:18]: Container component with header prop

**Review by lasharela — APPROVED** [review]

**Comment by Nap1400** [comment]
image

**Review by Josetf2396 — APPROVED** [review]

**Review by mshago — APPROVED** [review]

**Review by Josetf2396 — APPROVED** [review]

**Review by Dulaya — APPROVED** [review]

**Pull request merged**